### PR TITLE
Remove DisplayName, Description, and Category from non-visible properties

### PR DESCRIPTION
**Pull request by etbyrd**

### CODE REVIEW

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml [hunk 1]
```diff
@@ -148,7 +148,7 @@
     <EnumValue Name="OnOutputUpdated" DisplayName="When the build updates the project output" />
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralConfiguredBrowseObject.xaml]: These display names can be removed too

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedCOMReference.xaml [hunk 2]
```diff
@@ -86,21 +78,16 @@
...
     <StringProperty Name="RuntimeVersion"
                     ReadOnly="True"
-                    Visible="False"
-                    DisplayName="Runtime Version"
-                    Description="Version of the .NET runtime this assembly was compiled against.">
+                    Visible="False">
     </StringProperty>
 
     <BoolProperty Name="SpecificVersion" 
                   DisplayName="Specific Version"
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedCOMReference.xaml]: Missed one :P

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedPackageReference.xaml [hunk 3]
```diff
@@ -116,7 +114,6 @@
     <StringListProperty Name="Dependencies"
                         DisplayName="Dependencies"
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedPackageReference.xaml]: Can be removed

File: src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedProjectReference.xaml [hunk 4]
```diff
@@ -135,47 +133,38 @@
...
-                    DisplayName="Runtime Version"
-                    Description="Version of the .NET runtime this assembly was compiled against.">
+                    Visible="False">
     </StringProperty>
 
     <StringProperty Name="SDKIdentity"
                     Visible="false" />
 
     <BoolProperty Name="SpecificVersion" 
                   DisplayName="Specific Version"
```
Comment (davidwengier) [inline src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/ResolvedProjectReference.xaml]: Can be removed

**Comment by etbyrd** [comment]
My bad, I will do a second pass.

**Comment by etbyrd** [comment]
I believe I removed all of them.

**Pull request merged**